src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Truck started its work.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck stopped working.");

```

Comment:
Similarly, make the stop message more informative to indicate that it is the Truck that has stopped working.